packages/breadcrumbs/src/elements/Breadcrumb.tsx

```diff
@@ -37,18 +40,29 @@ const renderItems = (items, getCurrentPageProps) => {
   });
 };
 
+export interface IBreadCrumbProps extends React.HTMLProps<HTMLDivElement> {
+  children?: React.ReactNode;
```

Comment:
The `children` prop should be created for us by the `React.FC` call. This interface may not be needed.